cmd/defaults.yaml

```diff
@@ -824,8 +828,12 @@ DefaultInstance:
       Greeting: Hello {{.DisplayName}},
       Text: The password of your user has changed. If this change was not done by you, please be advised to immediately reset your password.
       ButtonText: Login
+  
+  # See SystemDefaults.Features for all possible values.
+  # Once a feature is set on the instance (true or false), system level feature settings
+  # will be ignored untill instance level features are reset.
   Features:
-    - FeatureLoginDefaultOrg: true
+    LoginDefaultOrg: true # ZITADEL_DEFAULTINSTANCE_FEATURES_LOGINDEFAULTORG
```

Comment:
The breaking changes were mentioned in the [PR description](https://github.com/zitadel/zitadel/pull/7356#issue-2123389334) under the "Breaking config changes" header, so that the person creating the release can copy-paste that into the release notes.